README.md

```diff
@@ -495,6 +498,7 @@ Option name          | Default value    | Description
 `:auto_paginate`     | `true`           | [Whether data should be fetched in batches or as one response](#fetching-methods)                 | YES    | YES |
 `:transaction_per_page` | `false`       | [Whether transaction should be per page of fetched objects or for all the pages - note that setting this value to `true` will mean always fetching data in batches, even when specifying `auto_paginate` as true](#persisting-fetched-objects)                 | YES    | YES |
 `:handle_processed_objects_proc` | `nil` | [Custom proc taking persisted remote objects, called after persisting batch of data](#persisted-objects)   | YES    | NO |
+`:model_name` | `nil` | name of syncronizeable model on Core | YES    | NO |
```

Comment:
is default value nil or name of current model? Also would avoid using Core in favour of BookingSync. Core sounds more like our internal concept.